Wolox/WoloxAndroidBootstrap
# product variants & flavor resources

**Summary**
1. Added product variants (flavors & builds). Also added specific resources (Configuration classes and icons) to each.
2. Changed _WoloxApplication_ code to retreive instance and extracted URLs to Configuration classes

**Test Plan**
1. Choose a product variant using the _stage_ flavor (_stageDebug_), build the app and see that it installs and opens correctly
2. Repeat for a _production_ flavor
3. Check the icons and app name/version. They should all differ between both variants
